test/0.8.9/staking-router/report-exited-keys.test.js

```diff
@@ -26,6 +26,9 @@ contract('StakingRouter', ([admin, depositor]) => {
     })
 
     router = deployed.stakingRouter
+    router.updateExitedValidatorsCountByStakingModuleWithResult = contractMethodWithResult(
```

Comment:
Modifying function names makes reading tests harder since it's not clear which underlying function is called.

Can we do smth like this instead?

```js
// in utils

function addResultHandler(method) {
  method.transactAndGetResult = async (...args) => {
    const result = await method.call(...args)
    await method(...args)
    return result
  }
}

// in the before handler

addResultHandler(router.updateExitedValidatorsCountByStakingModule)

// in a test

const result = await router
  .updateExitedValidatorsCountByStakingModule
  .transactAndGetResult(...args)
```

This way, it's clear which function is invoked, and that it's involked in some magical way.